distribution/openrct2.d.ts

```diff
@@ -1819,6 +1821,45 @@ declare global {
         companyValueRecord: number;
     }
 
+    type ClimateType =
+        "coolAndWet" |
+        "warm" |
+        "hotAndDry" |
+        "cold";
+
+    type WeatherType =
+        "sunny" |
+        "partiallyCloudy" |
+        "cloudy" |
+        "rain" |
+        "heavyRain" |
+        "thunder" |
+        "snow" |
+        "heavySnow" |
+        "blizzard";
+
+    interface ClimateState {
+        readonly weather: WeatherType;
+        readonly temperature: number;
+    }
+
+    interface Climate {
+        /**
+         * The climate of the park.
+         */
+        readonly climate: ClimateType;
+
+        /**
+         * The current weather in the park.
+         */
+        readonly currentWeather: ClimateState;
```

Comment:
I think this should just be `current`, otherwise you end up writing `currentWeather.weather`.